hs_core/tests/api/rest/test_resource_science_meta.py

```diff
@@ -0,0 +1,97 @@
+# import json
+
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+from .base import HSRESTTestCase
+
+
+class TestResourceScienceMetadata(HSRESTTestCase):
+
+    def setUp(self):
+        super(TestResourceScienceMetadata, self).setUp()
+
+        self.rtype = 'GenericResource'
+        self.title = 'My Test resource'
+        res = resource.create_resource(self.rtype,
+                                       self.user,
+                                       self.title)
+        self.pid = res.short_id
+        self.resources_to_delete.append(self.pid)
+
+    def test_get_scimeta(self):
+        # Get the resource system metadata
+        sysmeta_url = "/hsapi/resource/{res_id}/scimeta/elements/".format(res_id=self.pid)
+        response = self.client.get(sysmeta_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # content = json.loads(response.content)
+
+    def test_put_scimeta(self):
+        sysmeta_url = "/hsapi/resource/{res_id}/scimeta/elements/".format(res_id=self.pid)
+        put_data = {
+            "title": "New Title",
+            "description": "New Description",
+            "subjects": [
+                {"value": "subject1"},
+                {"value": "subject2"},
+                {"value": "subject3"}
+            ],
+            "contributors": [{
+                "name": "Test Name 1",
+                "description": "Desc 1",
+                "organization": "Org 1",
+                "email": "email1@email.com",
+                "address": "Address 1",
+                "phone": "1234567890",
+                "homepage": "http://homepage1.com"
+            }, {
+                "name": "Test Name 2",
+                "description": "Desc 2",
+                "organization": "Org 2",
+                "email": "email2@email.com",
+                "address": "Address 2",
+                "phone": "1234567891",
+                "homepage": "http://homepage2.com"
+            }],
```

Comment:
It would be good add a creator with data only for the required fields. I think in case of creator the required fields are either ```name``` or ```organization```. That means it should be possible to creator as long as there is data for one of these 2 fields.